rudder-core/src/test/scala/com/normation/rudder/services/policies/write/WriteSystemTechniquesTest.scala

```diff
@@ -188,6 +193,10 @@ object TestSystemData {
     //denybadclocks and skipIdentify are runtime properties
     , getDenyBadClocks         = () => Full(true)
     , getSkipIdentify          = () => Full(false)
+      // relay synchronisation method
+    , getSyncMethod            = () => Full(ClassicSynchronization)
+    , getSyncPromises          = () => Full(false)
```

Comment:
Does it makes sense (from the feature behavior) to have separated items for sync method / sync promises / sync shared file ? The two last seems to have a meaning only for syncMethod=rsync, no ? (really asking, I don't have any idea from the ticket description about what should be the behavior)

Reply:
for the moment they have, but it could change in the future - to have rsync for one, and classic for another for instance
I didn't want to go the json path for something that were bound to change , and I prefered to stricly match the form and the system variable